core/controllers/home.py

```diff
@@ -226,22 +226,9 @@ class NewExploration(base.BaseHandler):
     @base.require_fully_signed_up
     def post(self):
         """Handles POST requests."""
-        title = self.payload.get('title')
-        category = self.payload.get('category')
-        objective = self.payload.get('objective')
-        language_code = self.payload.get('language_code')
-
-        if not title:
-            raise self.InvalidInputException('No title supplied.')
-        if not category:
-            raise self.InvalidInputException('No category chosen.')
-        if not language_code:
-            raise self.InvalidInputException('No language chosen.')
-
         new_exploration_id = exp_services.get_new_exploration_id()
         exploration = exp_domain.Exploration.create_default_exploration(
-            new_exploration_id, title, category,
-            objective=objective, language_code=language_code)
+            new_exploration_id, '', '')
```

Comment:
It it has a default, would we have to change the ng-ifs that are looking at whether or not something is displayed there?